monitoring/ceph-mixin/tests_alerts/test_alerts.yml

```diff
@@ -142,8 +145,8 @@ tests:
        eval_time: 1m
        exp_samples:
          - labels: '{ceph_daemon="osd.0", hostname="ceph", instance="ceph:9283",
-           job="ceph"}'
-           value: 1.2200000000000001E+01
```

Comment:
@frittentheke , I think the error in make-check is because of this change. Can you please try reverting this change and check if the tests pass

Reply:
@frittentheke ^^